session.go

```diff
@@ -274,18 +273,6 @@ func (s *session) SetCronPeriod(period int) {
 	s.period = time.Duration(period) * time.Millisecond
 }
 
-// set @session's read queue size
-func (s *session) SetRQLen(readQLen int) {
```

Comment:
pls do not delete it. just leave a empty func body is enough to compatible with previous version.

Reply:
done